castai/resource_rebalancing_schedule.go

```diff
@@ -294,7 +304,12 @@ func scheduleToState(schedule *sdk.ScheduledrebalancingV1RebalancingSchedule, d
 
 	if schedule.LaunchConfiguration.RebalancingOptions != nil {
 		launchConfig["rebalancing_min_nodes"] = schedule.LaunchConfiguration.RebalancingOptions.MinNodes
-		launchConfig["keep_drain_timeout_nodes"] = schedule.LaunchConfiguration.RebalancingOptions.KeepDrainTimeoutNodes
+
+		if _, ok := d.GetOk("launch_configuration.keep_drain_timeout_nodes"); ok {
```

Comment:
not even sure how to implement this part: if user uses the old field, the fill the old field, otherwise use the new field.